### PR TITLE
Fix jEssentials script error in 3.3.3

### PR DESCRIPTION
**Pull request by HaSa1002**

### CODE REVIEW

**Comment by Jean28518** [comment]
I don't have any Error in GoDot 3.3.3

grafik

What error message do you get?

**Comment by HaSa1002** [comment]
`delayed_call_table.delay` does not exist.

**Comment by Jean28518** [comment]
When do you get the error message?

**Comment by HaSa1002** [comment]
Always

**Comment by HaSa1002** [comment]
but preferably in editor

**Comment by Jean28518** [comment]
Sorry can't reproduce that. What does your code changes actually fix? 
You just moved around some things. On which line in code do you get the error?

File: src/addons/jean28518.jTools/jEssentials/jEssentials.gd [hunk 1]
```diff
@@ -2,6 +2,9 @@ tool
 extends Node
 
 
+var delayed_call_table := {"delay" : [], "object" : [], "method" : [], "arg_array" : [] }
```
Comment (HaSa1002) [inline src/addons/jean28518.jTools/jEssentials/jEssentials.gd:5]: I initialise that variable at "compile time"

Comment (Jean28518) [reply src/addons/jean28518.jTools/jEssentials/jEssentials.gd:5]: Okay. Why not :)

Why do you make this public? (removing the '_' at the front)

Comment (HaSa1002) [reply src/addons/jean28518.jTools/jEssentials/jEssentials.gd:5]: it was before 🤷 
https://github.com/Jean28518/Libre-TrainSim/pull/139/files#diff-280a6b2040f6ccad3cfc706203c9edf89c1001ab399a06ef5da76c96130a055dL124

Comment (HaSa1002) [reply src/addons/jean28518.jTools/jEssentials/jEssentials.gd:5]: But we can change that ^^

Comment (Jean28518) [reply src/addons/jean28518.jTools/jEssentials/jEssentials.gd:5]: My fault, sorry!

**Pull request merged**